gtfs/spec/en/reference.md

```diff
@@ -254,17 +259,23 @@ Primary key (`trip_id`, `stop_sequence`)
+|  `arrival_time` | Time | **Conditionally Required** | Arrival time at the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`). <br><br>If there are not separate times for arrival and departure at a stop, `arrival_time` and `departure_time` should be the same. <br><br>For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins.<br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for the first and last stop in a trip (defined by `stop_times.stop_sequence`). <br>- **Required** for `timepoint=1`.<br>-&nbsp;**Forbidden** when `stop_times.start_pickup_drop_off_window` or `stop_times.end_pickup_drop_off_window` are defined.<br>- Optional otherwise.|
+|  `departure_time` | Time | **Conditionally Required** | Departure time from the stop (defined by `stop_times.stop_id`) for a specific trip (defined by `stop_times.trip_id`).<br><br>If there are not separate times for arrival and departure at a stop, `arrival_time` and `departure_time` should be the same. <br><br>For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins.<br><br> If exact arrival and departure times (`timepoint=1` or empty) are not available, estimated or interpolated arrival and departure times (`timepoint=0`) should be provided.<br><br>Conditionally Required:<br>- **Required** for `timepoint=1`.<br>-&nbsp;**Forbidden** when `stop_times.start_pickup_drop_off_window` or `stop_times.end_pickup_drop_off_window` are defined.<br>- Optional otherwise.| |
+|  `stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies the serviced stop. All stops serviced during a trip must have a record in [stop_times.txt](#stop_timestxt). Referenced locations must be stops/platforms, i.e. their `stops.location_type` value must be `0` or empty. A stop may be serviced multiple times in the same trip, and multiple trips and routes may service the same stop.<br><br>On-demand service within locations should be referenced in the sequence in which service is available in those locations. A data consumer should assume that a rider picked up at one stop or in one location can be dropped off at any stop or in any location later in the trip, pursuant to the `pickup/drop_off_type` of those stop_times and the time constraints of each `start/end_pickup_drop_off_window`.<br><br>Conditionally Required:<br>- **Required** if `stop_times.location_group_id` AND `stop_times.location_id` are NOT defined.<br>- **Forbidden** if `stop_times.location_group_id` or `stop_times.location_id` are defined. |
```

Comment:
I recommend changing "On-demand service within locations should be referenced in the sequence in which service is available in those locations."

to

"On-demand service using stops should be referenced in the sequence in which service is available at those stops."

Reply:
Commit pushed - https://github.com/google/transit/pull/388/commits/e428c8e75cda4e0b07a446a44acc5b0c92a2664d
Thanks!